src/Illuminate/Database/Seeder.php

```diff
@@ -120,18 +122,19 @@ public function setCommand(Command $command)
     /**
      * Run the database seeds.
      *
+     * @param  mixed ...$params
      * @return mixed
      *
      * @throws \InvalidArgumentException
      */
-    public function __invoke()
+    public function __invoke(...$params)
     {
         if (! method_exists($this, 'run')) {
             throw new InvalidArgumentException('Method [run] missing from '.get_class($this));
         }
 
         return isset($this->container)
-                    ? $this->container->call([$this, 'run'])
-                    : $this->run();
+                    ? $this->container->call([$this, 'run'], $params)
+                    : call_user_func_array([$this, 'run'], $params);
```

Comment:
`$this->run(...$params)`

Reply:
Should I do the same for `call_user_func_array([$seeder, '__invoke'], $params);` ?
`$seeder->__invoke(...$params)`

Reply:
Don't change any unrelated code to your PR.

Reply:
It's in my PR, I changed it same as this

Reply:
Yes, then. Please change it too.